plugins/woocommerce-blocks/assets/js/blocks/checkout/inner-blocks/checkout-shipping-method-block/style.scss

```diff
@@ -34,6 +35,11 @@
 	&.wc-block-checkout__shipping-method-option--selected {
 		outline: 1px solid $universal-border-strong;
 		background-color: $universal-background;
+
+		&:focus {
```

Comment:
all focus and hover styles are already being overridden so I have not made something **less** accessible with these changes imo, this just ensures they are specific enough in the `selected` to not change the appearance from the existing implementation.

Reply:
There are some changes to the styles in this PR https://github.com/woocommerce/woocommerce/issues/46008. Maybe we should wait for that to be merged?

Reply:
Hi @ralucaStan just to add more info - the PR you're referring to is #46150 and it is already merged.

I rebased this PR locally on trunk and it works fine with the changes from that PR 👌🏼

Reply:
Thanks for testing that @opr 🙇🏻

Reply:
Ah my bad for the misunderstanding, sorry! Ok then it _does_ make sense to wait for that to be merged, or rebase locally and see if it works.

Thanks for clarifying this @nielslange 🙇🏼

Reply:
@nielslange I agree probably to merge this PR first. If you need any help to test the changes in your PR later, please feel free to loop me in.